src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -44,34 +72,36 @@ TopicDomainHeader.propTypes = {
   handleOnMemberPress: PropTypes.func,
   hideSeeMember: PropTypes.bool,
   isFollow: PropTypes.bool,
-  memberCount: PropTypes.number
+  memberCount: PropTypes.number,
+  isLoading: PropTypes.bool
 };
 
 const styles = StyleSheet.create({
-  domainText: {
+  domainText: (isHeaderHide) => ({
     fontSize: normalizeFontSize(16),
     fontFamily: fonts.inter[600],
     textAlign: 'left',
-    color: colors.black,
+    color: isHeaderHide ? colors.white : colors.black,
     backgroundColor: 'transparent'
-  },
+  }),
   member: {
     width: normalize(16),
     height: normalize(16),
     marginRight: normalize(5)
   },
-  domainMember: {
+  domainMember: (isHeaderHide) => ({
     fontSize: normalizeFontSize(12),
     fontFamily: fonts.inter[400],
     textAlign: 'left',
-    color: colors.blackgrey
-  },
-  seeMemberText: {
+    color: isHeaderHide ? colors.white : colors.blackgrey
+  }),
+  seeMemberText: () => ({
     fontSize: normalizeFontSize(12),
     fontFamily: fonts.inter[500],
     textAlign: 'left',
-    color: colors.blue1
-  }
+    color: colors.blue,
+    marginTop: normalize(1)
+  })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the new hunk, you've added a new prop `isLoading` to the PropTypes of the component. However, it seems like you haven't provided a default value for this prop in case it's not passed from the parent component. This could potentially lead to unexpected behavior if `isLoading` is undefined. I suggest adding a default value for `isLoading` in `TopicDomainHeader.defaultProps`.

```diff
+ TopicDomainHeader.defaultProps = {
+   isLoading: false,
+ };
```
This way, even if the parent component doesn't pass the `isLoading` prop, your component will still have a defined value for it.


<!-- This is an auto-generated comment by OSS CodeRabbit -->